amqp/transport.py

```diff
@@ -54,7 +55,7 @@ class _AbstractTransport(object):
     """Common superclass for TCP and SSL transports"""
     connected = False
 
-    def __init__(self, host, connect_timeout):
+    def __init__(self, host, connect_timeout, write_timeout, read_timeout):
```

Comment:
@FlaPer87 i was under the impression that most consumers of this lib were using the connection class instead of the transport classes directly, i'll look into updating this pr to make those optional arguments